inference/models/owlv2/owlv2.py

```diff
@@ -614,3 +672,138 @@ def make_response(self, predictions, image_sizes, class_names):
             for ind, batch_predictions in enumerate(predictions)
         ]
         return responses
+
+
+class SerializedOwlV2(RoboflowInferenceModel):
+    task_type = "object-detection"
+    box_format = "xywh"
+
+    @classmethod
+    def serialize_training_data(
```

Comment:
do you think it's better to return the buffer in memory (instead of saving to a fixed path on /tmp to make this call thread-safe?)